src/main/scala/com/evolutiongaming/conhub/ConStates.scala

```diff
@@ -26,9 +24,9 @@ trait ConStates[Id, T, M] extends ConnTypes[T, M] {
 
   def update(id: Id, version: Version, conn: T, address: Address): Result
 
-  def disconnect(id: Id, version: Version, timeout: FiniteDuration, ctx: Ctx = Ctx.Local): Result
+  def disconnect(id: Id, version: Version, timeout: FiniteDuration, ctx: ConStates.Ctx = ConStates.Ctx.Local): Result
 
-  def remove(id: Id, version: Version, ctx: Ctx = Ctx.Local): Result
+  def remove(id: Id, version: Version, ctx: ConStates.Ctx = ConStates.Ctx.Local): Result
```

Comment:
I don't think this adds readability

Reply:
I've removed `import ConStates._` as Coveralls doesn't allow it.